src/index.html

```diff
@@ -22,6 +22,357 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo-container">
+        <a
+          class="header__logo"
+          href="#"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link header__link--is-active is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link"
+              data-qa="nav-hover"
```

Comment:
Ensure that 'data-qa' attributes are used consistently and only for testing purposes. If this attribute is used to identify elements for styling or scripting, consider using classes or IDs instead.